code/modules/jobs/job_types/security.dm

```diff
@@ -101,7 +101,7 @@ Warden
 	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
 	r_pocket = /obj/item/device/assembly/flash/handheld
 	l_pocket = /obj/item/weapon/restraints/handcuffs
-	suit_store = /obj/item/weapon/gun/energy/e_gun/advtaser
+	suit_store = /obj/item/weapon/gun/energy/frame/hybrid_taser

```

Comment:
this from the guy who takes pride in not playing unless it's one of his testmerges. i haven't been around for your testmerges, jerk